MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1168 @@
+
+{% tabs %}
+{% highlight c# %}
+
+    this.dataForm.RegisterEditor("On", DataFormEditorType.Switch);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+
+Here, the Switch editor will be loaded for the `On` property (bool type) instead of `CheckBox` editor.
+
+## Text editor
+
+In the text editor, the [Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0) is loaded.
+
+Text editor have Keyboard type change by using `Keyboard` property and you can set maximum allowed length of input by using `MaxLength` property.
+
+## Multiline Text editor
+
+In the `MultilineText` editor, the [Editor](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-7.0) is loaded.
+
+And `MultilineText` editor height will auto expand/reduce based on the line wraps in editor , which allowing text to be readable without scrolling the editor.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.MultilineText)]
+    public string Address { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Password editor
+
+In the password editor, the [Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0) is loaded.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Display(ShortName = "Transaction password", Prompt = "Enter password")]
+    [DataType(DataType.Password)]
+    public string Password { get; set; }
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Date editor
+
+In the date editor, the [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0) will be loaded.
+
+In [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0), the default date value is current date displayed. You can also adding nullable `DateTime` data type for the date picker property in data form, which allows you to set the current date and display current date in date editor. 
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Date)]
+    [Display(Name ="Event Date")]
+    public DateTime? EventDate { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Customizing format in date editor
+
+In the [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0), short date will be shown by default. You can change the applied format by setting the [Format]`Link`property in [DataFormDateItem]`Link`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {   
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date" && e.DataFormItem is DataFormDateItem dateItem)
+        {
+            dateItem.Format = "dd, MM, yyyy";
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Setting MaximumDate and MinimumDate in date editor
```

Comment:
### Change the MaximumDate and MinimumDate in date editor

Reply:
updated